src/mantine-core/src/Rating/StarSymbol/StarSymbol.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { createStyles, MantineSize } from '@mantine/styles';
+
+export interface StarSymbolProps {
+  size?: MantineSize;
+  type?: 'empty' | 'full';
+}
+
+export default createStyles<'empty' | 'full'>((theme) => {
```

Comment:
Do not add generic params to createStyles, should be `createStyles((theme) => {`